sbc_compassion/models/import_letters_history.py

```diff
@@ -62,7 +54,7 @@ class ImportLettersHistory(models.Model):
         'import.letter.line', 'import_id', 'Files to process')
     letters_ids = fields.One2many(
         'sponsorship.correspondence', 'import_id', 'Imported letters',
-        readonly=True)
+        readonly=True, ondelete='cascade')
```

Comment:
sorry I was a little bit too fast ^^
I wanted to put it to the other one. I think that this one should be kept normal (not cascade) as it is possible to delete everything from the other menu